tests/schema_registry/test_proto.py

```diff
@@ -19,13 +19,41 @@
 from io import BytesIO
 
 import pytest
+from google.protobuf.message_factory import MessageFactory
 
 from confluent_kafka.schema_registry.protobuf import (ProtobufSerializer,
                                                       ProtobufDeserializer,
                                                       _create_msg_index)
+from confluent_kafka.serialization import SerializationContext, MessageField
 from tests.integration.schema_registry.data.proto import (DependencyTestProto_pb2,
-                                                          metadata_proto_pb2)
+                                                          metadata_proto_pb2,
+                                                          TestProto_pb2,
+                                                          PublicTestProto_pb2,
+                                                          NestedTestProto_pb2)
+from tests.schema_registry.test_api_client import TEST_URL
 
+TEST_DATA = [
```

Comment:
No need for such complex data while testing our usecase. Keep only what is really required.

Move data near the testcase.